### DISCUSSION TITLE
ASP.NET MVC Web APP Github Actions Build

### DISCUSSION THREAD

**Discussion by bradmasy** [community/community#46299] [Actions]
### Select Topic Area

Product Feedback

### Body

Hey Everyone,
Im creating a CI Pipeline with github actions and fairly new to ASP.NET. I know the dotnet build command usually builds a project but I keep getting errors if I try this with my MVC web app. Anyone know the cmd command to build an ASP.NET MVC Web App so I can automate it on push?

Thanks!

**Comment by shutirtha-roy** [comment]
I think this [url ](https://www.youtube.com/watch?v=FBm3WypsXy4) can help you, if you are running it in Windows 11 OS.
